concepticondata/conceptlists/Verheyen-2019-1000.tsv

```diff
@@ -0,0 +1,1001 @@
+Verheyen-2019-1000-44	44	asociaal	antisocial			10.3	2.25	4.1	3.3	5.35	4.15	1.8	8	3	33308	0	106	65	1.34788407300923	0.000252638458632
+Verheyen-2019-1000-45	45	asymmetrisch	asymmetric			11.45	3.9	5.2	4.25	3.75	6.05	2.95	12	4	73048	0	40	12	0.012448132780083	4.16535360402665E-07
+Verheyen-2019-1000-46	46	avondlijk	evening			11.9285714285714	3.90909090909091	3.75	4.52941176470588	1.82352941176471	3.9	4.88235294117647	9	3	52515	0	50	1	0.004016064257028	3.1890988530829E-07
+Verheyen-2019-1000-47	47	baldadig	rowdy			12.9333333333333	5.5	3.46666666666667	4.88888888888889	1.70588235294118	3.0625	2.875	8	3	32201	0	74	2	0.033604275655867	6.21548858100851E-06
+Verheyen-2019-1000-48	48	bang	afraid	FEAR (BE AFRAID)	1419	3.95	4.35	4.05	2.75	5.75	5.9	2.3	4	1	32714	14	7921	21119	9.56742495937603	0.001036131709002
+Verheyen-2019-1000-49	49	bar	barren			8.8	3.68421052631579	4.29411764705882	3.05263157894737	3.73684210526316	5.65	3.9	3	1	14841	19	725	98	1.39625465570824	0.000294051185167
+Verheyen-2019-1000-50	50	barbaars	wild	UNTAMED OR WILD	2230	10.95	5.3	4.4	4.6	2.7	4.9	2.15	8	2	51975	0	292	153	0.102223583069015	2.93429636308658E-05
+Verheyen-2019-1000-51	51	bazig	overbearing			6.75	5.4	4.15	5.95	3.45	5.1	2.45	5	2	13388	6	59	79	0.83775508875755	0.000130902745372
+Verheyen-2019-1000-52	52	bebost	forested			9.1578947368421	3.47058823529412	5.38888888888889	4.16666666666667	2.25	6.55	3.75	6	2	35893	1	0	0	0.012833031758907	3.64468440352332E-06
+Verheyen-2019-1000-53	53	bebouwd	built-on			8.65	4.33333333333333	4.9	4.5	3.55	6.15	3.05	7	2	20186	1	80	3	0.024403700874289	2.23887756216432E-06
+Verheyen-2019-1000-54	54	bedaard	cool			10	2.33333333333333	3.11111111111111	3	2.75	3.78947368421053	4.1	7	2	46985	4	199	4	0.087074744532016	5.73386957054293E-06
+Verheyen-2019-1000-55	55	bedekt	covered			7.2	3	3.94736842105263	3.25	3.6	5.4	4.1	6	2	35281	1	849	1	0.226788101817662	7.67010815991469E-05
+Verheyen-2019-1000-56	56	bedrukt	dejected			10	3.75	4.15	3.95	2.95	5.05	2.5	7	2	22872	2	127	37	0.198639169914251	7.37202504262654E-05
+Verheyen-2019-1000-57	57	befaamd	famous			11.3	4.55555555555556	3.76470588235294	4.94736842105263	2.4	4.1	4.85	7	2	31316	1	431	58	0.062929194145114	5.4832975178007E-06
+Verheyen-2019-1000-58	58	begaafd	gifted			10.1	4.55	4.25	5.3	4.05	3.3	4.75	7	2	34276	0	525	173	0.566854407163204	8.90409416510759E-05
+Verheyen-2019-1000-59	59	begrijpelijk	understandable			8.05	3.65	3.9	3.85	5.55	3.85	4.85	12	4	91621	0	1239	240	0.345776124977515	4.79243457238285E-05
+Verheyen-2019-1000-60	60	beige	beige	BEIGE	3707	7.8	2.35	5.9	2.75	3.3	6.55	3.4	5	2	37743	1	191	73	0.700583079428269	0.000139002405622
+Verheyen-2019-1000-61	61	bejaard	elderly	OLD (AGED)	2112	7.4	2	5.3	3	4.4	6.75	3.85	7	2	40344	4	288	84	1.01741566986598	0.000113876862515
+Verheyen-2019-1000-62	62	beknopt	short	SHORT	1645	10.75	3.8	4.45	3.2	4.05	3.95	4	7	2	23060	0	238	17	0.572122035398519	5.01697316509991E-05
```

Comment:
beknopt has a meaning similar to concise and is used for ideas, texts etc., not in the spatial sense, so the concept does not fit